docs/poc-guidlines.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: Firebolt POC Guidelines
+description: Make the necessary preparations for your Firebolt PoC
+nav_order: 12
+---
+
+# Firebolt POC Guidelines
+
+## Thank you for considering Firebolt. We created this document to help both our teams work better together during the POC.
+
+### **Accessing data in S3**
+
+The data that will be used in the POC should be made available in an S3 region (any region is fine). Don’t hesitate to make all or a very large portion of the data available – the more data the better.
+
+The following policy should be granted to the S3 bucket:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::231290928314:root"
+      },
+      "Action": [
+        "s3:GetObject",
+        "s3:GetObjectTagging",
+        "s3:PutObject",
+        "s3:PutObjectTagging"
+        "s3:ListBucket",
+        "s3:GetBucketLocation"
+      ],
+      "Resource": [
+        "arn:aws:s3:::<MY-BUCKET>",
+        "arn:aws:s3:::<MY-BUCKET>/*"
+      ]
+    }
+  ]
+}
+```
+
+To alter the S3 bucket policy:
+
+1. Connect to AWS console and click on the relevant S3 bucket
+2. Go to "Permissions" and then scroll down and edit "Bucket Policy"
+3. Copy the policy mentioned above, make sure you replace "MY-BUCKET"
+with the actual bucket name
+4. Save changes
+
+<aside>
```

Comment:
Let's move this aside above the instructions - customers like to know why they are doing something before they do it

Reply:
Done